charts/metal3/0.2.0/charts/ironic/values.yaml

```diff
@@ -88,15 +87,30 @@ podAnnotations: {}
 podSecurityContext: {}
   # fsGroup: 2000
 
+requireProvisioningIP: true
+
+# Host on which ironic can reach its database.
+# If left empty, host will be resolved to the FQDN
+# of the service with name provided under the 'global.databaseServiceName' property
+databaseHost: ""
```

Comment:
Did you have a specific use-case in mind for this?  IMO it can probably be removed, since we don't expect to support an external DB - and it's possible we may want to support disabling mariadb (instead using sqlite in the ironic container) which will be easier if there are no user-accessible customisations related to the DB connection.

Reply:
The use-case in mind I had was if we decide to switch to another DB, but if there will be no such use-case I will remove it.